pillarbox-demo-tv/src/main/java/ch/srgssr/pillarbox/demo/tv/ui/integrationLayer/SearchView.kt

```diff
@@ -191,11 +192,21 @@ private fun SearchInput(
 private fun NoResults(
     modifier: Modifier = Modifier
 ) {
-    Box(
+    Column(
```

Comment:
Maybe we could merge `NoResults` and `NoContent` together, and padd the `imageVector` and `text` as arguments?
Same in the other demo

Reply:
I think the same.